requirements.in

```diff
@@ -7,7 +7,3 @@ python-dotenv
 pydantic
 orjson
 httpx
-# sse-stuff
-sse-starlette
-asyncio
-redis
```

Comment:
I guess we may want to keep supporting redis as a central cache for the edge proxy... but this is probably something we can handle later down the line.

Reply:
Yeah, it's just in memory right now